orchestrate.py

```diff
@@ -327,7 +332,9 @@ def main():
 
     handlers = [
         (r"/api/spawn/?", APISpawnHandler),
-        (r"/api/stats/?", APIStatsHandler)
+        (r"/api/stats/?", APIStatsHandler),
+        (r"/stats/?", RedirectHandler, {"url": "/api/stats"}),
+        (r"/info/?", InfoHandler),
```

Comment:
Ah. It's true that the data won't be available. But I thought `/info` exposes a whole UI for viewing the stats. Won't that be broken / useless now since it can't get to the data without the API token?